CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # bedrock-authn-token-http ChangeLog
 
+## 2.0.0 - 2021-01-TBD
+
+### Changed
+- **BREAKING**: Update peerDependencies to include bedrock-account@5 and bedrock@4.
```

Comment:
Can more be said about how/why upgrading the bedrock-account@5 constitutes a breaking change here?  I just did a quick pass and I don't see any new/different bedrock-account API calls.  And likewise, given that this module was not using any exotic logging features, the bedrock@4 bump does not make for a breaking change.

Folks internally and externally reading changelogs need to be able to assess what is going to be involved in upgrading from v1 to v2 here. So, we ask ourselves, if one wrote an application using v1, what is going to break if I simply change the dependency to v2?

A major bump in a peer dependency does not *necessarily* mean a breaking change here *unless* the breaking change in the peer dependency is somehow manifest in a public API change here.

Reply:
okay, I think I understand, changing this to a minor release.